microsoft-edge/devtools-guide-chromium/javascript/snippets.md

```diff
@@ -22,149 +22,158 @@ ms.date: 05/04/2021
 
-Snippets are an alternative to [bookmarklets](https://wikipedia.org/wiki/Bookmarklet) with the difference that Snippets only run in DevTools and aren't limited to the allowed length of a URL.
+The following screenshot shows Microsoft Edge with a webpage on the left, and DevTools on the right. The **Sources** tool is open showing the source code of a snippet. The snippet code was run, making changes to the webpage.
 
-Using Snippets is an excellent way to change a few things in a third-party webpage.  Code changes in Snippets are added to the current webpage and run in the same context.  For more information about changing the existing code of a webpage, see [Overrides](overrides.md).
+![The webpage, altered by the snippet](./snippets-images/snippets-overview.png)
 
-The following figure shows the DevTools homepage on the left and some Snippet source code on the right.
-
-The webpage before running the Snippet:
-
-![The webpage before running the Snippet](./snippets-images/javascript-sources-snippets-split-screen.png)
-
-The Snippet source code from the webpage before running the Snippet:
+The snippet source code is shown below:
 
 ```javascript
-console.log('Hello, Snippets!');
-document.body.innerHTML = '';
-var p = document.createElement('p');
-p.textContent = 'Hello, Snippets!';
+// Change the background color to "red".
+document.body.style.backgroundColor = "red";
+
+// Add a paragraph at the bottom of the document.
+const p = document.createElement("p");
+p.textContent = "Hello world";
+p.style.color = "white";
+p.style.fontSize = "2rem";
 document.body.appendChild(p);
+
+// Log a message to the console.
+console.log("Hello world");
 ```
 
-In the following figure, the webpage appears after running the Snippet.  The **Console Drawer** pops up to display the **Hello, Snippets!** message that the Snippet logs, and the content of the webpage changes completely:
+The code changes the background color of the webpage to red, adds a new line of text at the bottom of the webpage, and logs a message to the **Console** tool.
 
-![The webpage after running the Snippet](./snippets-images/javascript-sources-snippets-split-screen-after.png)
+When you run a snippet on a webpage, the snippet's source code is added to the current webpage. For more information about changing the existing code of a webpage, instead of adding new code, see [Override webpage resources with local copies (Overrides tab)](./overrides.md).
+
+**Note:** The security settings of most webpages block from loading other scripts in snippets. For this reason, you must include all your code in one file.
```

Comment:
```suggestion

#### Include all your code in one file

The security settings of most webpages block from loading other scripts in snippets. For this reason, you must include all your code in one file.
```

The unstructured "Note: " construction is usually better replaced by an h4 serving like a skimmable "label" on the paragraph.  Heavy use of "Note" is a symptom of needing structured headings.